app/src/main/java/com/amaze/filemanager/fragments/ZipViewer.java

```diff
@@ -78,9 +76,19 @@
 import java.util.ArrayList;
 import java.util.Calendar;
 
-public class ZipViewer extends Fragment {
+public class ZipViewer extends Fragment implements BottomBarButtonPath {
```

Comment:
Yes, but I would have to change a lot of fragments to have their correct name and that would make a lot more changes and the pull more prone to conflict. That should be corrected in another pull request (with just naming corrections).